README_WINDOWS.md

```diff
@@ -1,261 +1,265 @@
-#  bitpit in Microsoft Windows
+#  bitpit on Microsoft Windows
```

Comment:
Please merge this commit with the one that introduces the README file.

Reply:
done